tests/contrib/django_util/test_views.py

```diff
@@ -169,9 +169,11 @@ def test_callback_works(self, pickle):
             redirect_uri=request.build_absolute_uri("oauth2/oauth2callback"))
 
         name = 'google_oauth2_flow_{0}'.format(self.CSRF_TOKEN)
-        self.session[name] = pickle.dumps(flow)
+        self.session[name] = jsonpickle.encode(flow)
+        self.session['google_oauth2_flow_{0}'.format(self.CSRF_TOKEN)] \
+            = jsonpickle.encode(flow)
```

Comment:
Please don't use backslash continuation (in the Google style guide) Instead:

```python
        key = 'google_oauth2_flow_{0}'.format(self.CSRF_TOKEN)
        self.session[key] = jsonpickle.encode(flow)
```

I just fixed this in #608, and you put it back :cry: